layers/+misc/nav-flash/config.el

```diff
@@ -0,0 +1,7 @@
+(defvar nav-flash-exclude-commands
```

Comment:
Need to add header
https://github.com/syl20bnr/spacemacs/blob/develop/CONTRIBUTING.org#file-header